test/Test/Data/Mutable/Array.hs

```diff
@@ -1,16 +1,112 @@
+{-# LANGUAGE LinearTypes #-}
 {-# LANGUAGE OverloadedStrings #-}
-{-# LANGUAGE TemplateHaskell #-}
+{-# OPTIONS_GHC -Wno-name-shadowing #-}
 
-module Test.Data.Mutable.Array (mutArrTests) where
+-- |
+-- Tests for mutable arrays.
+--
+-- TODO:
+--  * Test failures for out of bound access
+module Test.Data.Mutable.Array
+  ( mutArrTests,
+  )
+where
 
+import qualified Data.Array.Mutable.Linear as Arr
```

Comment:
```suggestion
import qualified Data.Array.Mutable.Linear as Array
```

Reply:
I think Arr is a common and clear abbreviation.

Reply:
To be clear: I'm not offering this one up for negotiation.

Reply:
In the future, I'll implement all suggested changes in a review of one of my PRs :)

Also, reconsidering it, shorter qualified names can get terrible if there are a lot of them.